src/main/resources/index.jelly

```diff
@@ -0,0 +1,5 @@
+<?jelly escape-by-default='true'?>
+<div>
+  Scan entire GitHub organization for buildable repositories and branches.
```

Comment:
I would add explicitly **pull requests** term, for example, *Scan entire GitHub Organization for buildable branches and pull requests, in repositories*. The sentence is not important but using the term *pull request*.